cmd/osctl/cmd/install_linux.go

```diff
@@ -40,46 +37,20 @@ var installCmd = &cobra.Command{
 			err    error
 		)
 
-		platform, err := platform.NewPlatform()
-		if err == nil {
-			if !strings.EqualFold(platform.Name(), platformArg) {
-				log.Printf("platform mismatch (%s != %s)", platform.Name(), platformArg)
-			} else {
-				var b []byte
-				b, err = platform.Configuration()
-				if err != nil {
-					log.Fatal(err)
-				}
-				var content machineconfig.Content
-				content, err = machineconfig.FromBytes(b)
-				if err != nil {
-					log.Fatal(err)
-				}
-				config, err = machineconfig.New(content)
-				if err != nil {
-					log.Fatal(err)
-				}
-				extraKernelArgs = append(extraKernelArgs, config.Machine().Install().ExtraKernelArgs()...)
-			}
-		}
```

Comment:
We shouldn't try to be smart here. We can handle this logic in `machined` and run this command with the proper args instead.